tools/autograd/gen_python_functions.py

```diff
@@ -701,9 +701,17 @@ def group_overloads(declarations, is_python_method):
     result = []
     for x, dictionary in sorted(grouped.items()):
         if 'base' not in dictionary:
+            candidates = []
+            non_out_name = dictionary['out']['operator_name']
+            for declaration in declarations:
+                if declaration['name'] == non_out_name and not declaration['deprecated']:
+                    signature = get_python_signature(declaration, is_python_method, skip_outputs=True)
+                    candidates.append(signature)
             raise RuntimeError(
-                "'base' not in dictionary for {}. keys are {}".format(
-                    x, list(dictionary.keys())))
+                "While identifying overloads, we found an out schema {} without a corresponding non-out variant. "
+                "We expected the non-out variant to have schema: \n- {}\nPlease check that you spelled the schema "
+                "correctly or in native_functions.yaml. We discovered the following candidate(s): \n"
+                .format(dictionary['signature'], x) + "\n".join("- {}".format(candidate) for candidate in candidates))
```

Comment:
Can you update the sample error message in the PR body?

Reply:
Sure. There is actually a typo in the error message, and I will fix it and update PR.